src/mol-canvas3d/canvas3d.ts

```diff
@@ -571,10 +571,10 @@ namespace Canvas3D {
             if (helper.debug.isEnabled) helper.debug.update();
             if (!p.camera.manualReset && (reprCount.value === 0 || shouldResetCamera())) {
                 cameraResetRequested = true;
+                camera.setState({ radiusMax: scene.boundingSphere.radius }, 0);
             }
             if (oldBoundingSphereVisible.radius === 0) nextCameraResetDuration = 0;
 
-            camera.setState({ radiusMax: scene.boundingSphere.radius }, 0);
             reprCount.next(reprRenderObjects.size);
```

Comment:
This changes the semantics and might break some stuff (I think the reason for this extra line was dealing with animating state).

Adding the condition in front of the original line

```ts
if (!p.camera.manualReset) camera.setState({ radiusMax: scene.boundingSphere.radius }, 0);
```

might do the trick better.